openfisca_core/tools/test_runner.py

```diff
@@ -212,14 +213,28 @@ def check_variable(self, variable_name, expected_value, period, entity_index = N
 
         actual_value = self.simulation.calculate(variable_name, period)
 
+        absolute_error_margin = self.test.get('absolute_error_margin')
+        variable_absolute_error_margin = (
+            absolute_error_margin.get(variable_name, absolute_error_margin.get("default"))
+            if isinstance(absolute_error_margin, collections.Mapping)
```

Comment:
Can't we rather assume that if not declared, relative error margin is 0%?

Reply:
You want to be able to easily (continuously) deviate from the current situation where you have a global error margin